buildpacks/typescript/lib/build.sh

```diff
@@ -27,14 +27,42 @@ clear_cache_on_stack_change() {
 	fi
 
 	if [[ ! -f "${layers_dir}/store.toml" ]]; then
+		# shellcheck disable=SC2005
+		curr_node_version="$(echo "$(node -v)")"
 		touch "${layers_dir}/store.toml"
 		cat <<TOML >"${layers_dir}/store.toml"
 [metadata]
 last_stack = "$CNB_STACK_ID"
+node_version = "$curr_node_version"
```

Comment:
I think this is leftover from the previous commits - also, like I mentioned in npm, I don't think you need this `if` statement anymore